mycli/packages/expanded.py

```diff
@@ -17,7 +28,8 @@ def expanded_table(rows, headers):
     header_len += 2
 
     for row in rows:
-        row_len = max([len(_text_type(x)) for x in row])
+        row = [format_field(x) for x in row]
+        row_len = max([len(x) for x in row])
```

Comment:
Ah! row is actually used directly below this code to present it to the user, not just to check the length. 
I figured I would just use row immediately before the length check and the output to hexify everything while making sure the length check. 

I could swap the list comprehension (or whatever python calls them) for a generator if you're concerned about memory usage, but I believe this would be at the cost of extra processing each time the generator is used.